canonical/ubuntu.com
# [WD-6778] disable 23.10 desktop download button

## Done

- Disabled 23.10 desktop image download button/link on two pages:
  - /download/desktop
  - /download/alternative-downloads
- Added a help text to highlight that it's unavailable

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigage to /download/desktop and check that the `Download 23.10` button is disabled and has a helper text below
- Navigate to /download/alternative-downloads and check that `Download 23.10 Desktop (64-bit)` link doesn't exist and there a helper text instead.

## Issue / Card

Closes https://warthogs.atlassian.net/browse/WD-6778

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
